helm_deploy/templates/_helpers.tpl

```diff
@@ -55,6 +55,18 @@ This label allows the allow-prometheus-scraping NetworkPolicy to identify pods t
 app: laa-crime-application-store-prometheus
 {{- end }}
 
+{{/*
+Default Security Context
+*/}}
+{{- define "helm_deploy.defaultSecurityContext" -}}
```

Comment:
for consistency it should be prefixed with the what other "methods" are prefixed with, namely 

```
{{- define "laa-crime-application-store.defaultSecurityContext" -}}
```